### PR TITLE
Remove double sidekiq config block

### PR DESCRIPTION
**Pull request by mejackreed**
While doing some sidekiq sleuthing I say this.

### CODE REVIEW

**Review by jcoyne — APPROVED** [review]
Thanks.

File: config/initializers/sidekiq.rb [hunk 1]
```diff
@@ -3,7 +3,3 @@
 Sidekiq.configure_server do |config|
   config.redis = { url: Settings.redis_url }
 end
-
-Sidekiq.configure_client do |config|
```
Comment (jcoyne) [inline config/initializers/sidekiq.rb:7]: This is for the client, and the other is for the server, so is it really double?

Comment (mjgiarlo) [reply config/initializers/sidekiq.rb:7]: @mejackreed thoughts on this?

Comment (mejackreed) [reply config/initializers/sidekiq.rb:7]: Oops! I guess I missed that :)

**Pull request closed without merging**